release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -71,6 +71,8 @@ See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades
 phased roll out] for more information on the types of updates and upgrade
 channels.
 
+Because upgrades are published to the channels on a time delay as they are gradually rolled out to customers based on data from our SRE team, you might not immediately be able to update from version 4.1.z to 4.2 at initial release.
+
```

Comment:
Red Hat Service Reliability Engineering (SRE)

Reply:
I would lose "on a time delay" and just say "upgrades are gradually rolled out to customers". It's not a "time delay" at all, it's based on the data, period.

Reply:
(didn't see the updated commit, these current words are +1 from me)

Reply:
+1

Reply:
Need acks from @reestr, and possibly someone from the BU. Possibly @mpbarrett or @knewcomerRH .

I believe eng has given the ack via @cuppett (here) and @calfonso (via email).

Reply:
LGTM